odoo/addons/test_performance/models/models.py

```diff
@@ -67,3 +72,11 @@ class Eggs(models.Model):
     _description = 'Test Performance Eggs'
 
     name = fields.Char()
+
+
+class Mozzarella(models.Model):
+    _name = 'test_performance.mozzarella'
+    _description = 'Test Performance Mozzarella'
+
+    value = fields.Integer(default=0)
+    value_plus_one = fields.Integer(compute="_value_plus_one", default=1, store=True)
```

Comment:
compute + default?

Also "value_plus_one" seems to be in a different model?

(Not sure we need this to be complicated though, having 2 regular fields and incrementing them both in the test should be enough?)